maker/src/cfd.rs

```diff
@@ -367,6 +370,25 @@ where
             leverage,
         );
 
+        if let Err(e) = balance_check(
+            cfd.margin(),
+            order_to_take.tx_fee_rate,
+            &self.wallet,
+            &self.wallet,
+        )
+        .await
+        {
+            tracing::warn!("{e}");
```

Comment:
🔧 I don't think this would be redundant.

```suggestion
            tracing::warn!("Rejecting taker's order: {e}");
```